### DISCUSSION TITLE
<title> I can't upload PNG file

### DISCUSSION THREAD

**Discussion by kyungyeon-byun-imweb-me** [community/community#58503] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello.

I want to upload image to Pull Request.
So I made screenshot on macos.

i uploaded an image, but it is not uploading

the error message is 
`We don't support that file type. Try again with a GIF, JPEG, JPG, MOV, MP4, PNG, SVG .....`

The extension of the uploaded image file is PNG

What could be the problem?

**Comment by datpmt** [comment, marked as answer]
it may be your browser, can you try another browser?
https://github.com/orgs/community/discussions/47026#discussioncomment-4962795

**Reply by kyungyeon-byun-imweb-me** [reply]
thanks..! Reinstalled my chrome browser. and It works well.

**Reply by datpmt** [reply]
great! please make my answer the solution to this post. I'm looking for badge

**Reply by JennPlothow** [reply]
https://cowrite.me/v/2cc930a4-4ff5-4220-893e-6df926a273d4
